### PR TITLE
xcodes: 1.3.0 -> 1.4.1

### PR DESCRIPTION
**Pull request by 0x120581f**
###### Description of changes
[changelog](https://github.com/XcodesOrg/xcodes/releases/tag/1.4.1)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/xcodes/default.nix [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "xcodes";
-  version = "1.3.0";
+  version = "1.4.1";
 
   src = fetchurl {
     url = "https://github.com/XcodesOrg/xcodes/releases/download/${finalAttrs.version}/xcodes.zip";
-    hash = "sha256:0cqb0gfb80xrnm4fipr46kbzqz2kicc13afhdxkbifzm4k83any5";
+    hash = "sha256:1ff2ayx7hig1aamqpzg72gn2rdf6vp49amsbykv4az4dxbcwbm9y";
```
Comment (drupol) [inline pkgs/development/tools/xcodes/default.nix:9]:
```suggestion
    hash = "sha256-1ff2ayx7hig1aamqpzg72gn2rdf6vp49amsbykv4az4dxbcwbm9y";
```

**Review by drupol — CHANGES_REQUESTED** [review]
LGTM, a very minor thing to change to be aligned with most derivations in `nixpkgs` and it's good to go.

**Review by drupol — APPROVED** [review]

File: pkgs/development/tools/xcodes/default.nix (hunk 1 above)
Comment (drupol) [reply pkgs/development/tools/xcodes/default.nix]: Thanks!

**Comment by 0x120581f** [comment]
> Ping me when eval and tests are done and I'll merge it.

@drupol tests have finished successfully ✅

**Comment by drupol** [comment]
Thanks!

**Pull request merged**